locale/cs_CZ/common.xml

```diff
@@ -100,7 +100,7 @@
 	<message key="common.fileName">Jméno souboru</message>
 	<message key="common.fileSize">Velikost souboru</message>
 	<message key="common.fileType">Typ souboru</message>
-	<message key="common.file.namingPattern">{$username}, {$originalFilename}</message>
+	<message key="common.file.namingPattern">{$username}, {$userGroup}, {$originalFilename}</message>
```

Comment:
Should be `<message key="common.file.namingPattern">{$username}, {$originalFilename}</message>`

Reply:
Changed. Thank you.